### DISCUSSION TITLE
添加widget的切换按钮

### DISCUSSION THREAD

**Discussion by chenxiex** [CaiJimmy/hugo-theme-stack#753] [Ideas]
现在的widget是在小屏幕上自动关闭。但这个自动关闭的效果有时候不太理想，例如我的14寸2k屏幕就是恰好处在不关闭的边界上，但是实际效果看起来很挤。能否添加一个切换按钮，让大屏幕的用户也可以自行关闭或者开启widget？

**Comment by CaiJimmy** [comment]
抱歉，虽然是个很好的提议，但是目前没有精力再给主题添加新功能了。
